packages/core/src/hooks/useGasPrice.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from 'react'
+import { useEthers } from './useEthers'
+
+export function useGasPrice(pollingInterval?: number): number {
+  const { library } = useEthers()
+  const [gasPrice, setGasPrice] = useState(0)
+
+  useEffect(() => {
+    async function updateGasPrice() {
+      const newGasPrice: string = await library?.perform('getGasPrice', null)
```

Comment:
Why not `library?.getGasPrice()` and return a BigNumber instead of number

Reply:
Got error in my ide so I thought it wasn't provided but it shows up now. Updated